packages/services/storage/src/index.ts

```diff
@@ -636,33 +673,18 @@ export async function createStorage(
           );
         }
 
-        return action;
+        return {
+          user: internalUser,
```

Comment:
Related to https://github.com/graphql-hive/console/pull/7390/changes#r2635220421, wouldn't this potentially return the wrong user in a scenario where we have more than one user in our database with that email?

And causing the wrong `user.id` being used here:

<img width="695" height="422" alt="Image" src="https://github.com/user-attachments/assets/b6f701cd-47b7-4f51-b69c-7003efab0426" />

Reply:
Would https://github.com/graphql-hive/console/pull/7390/commits/4e0ab08212d64ce39587fdcb9aad2a050d41d0d7 be enough to resolve all of the issues you pointed out?